manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    """Run administrative tasks."""
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'movie_api.settings')
```

Comment:
The settings module path is incorrectly set to 'movie_api.settings'. It should be 'cinema.settings' to match the correct project name.